src/github.com/getlantern/flashlight/main/main.go

```diff
@@ -53,7 +54,12 @@ func init() {
 }
 
 func logPanic(msg string) {
-	cfg, err := config.Init(flashlight.PackageVersion, *configdir, *stickyConfig, flagsAsMap())
+	// Request the config via either chained servers or direct fronted servers.
+	// We have to do the full version here because we're actually trying to
+	// submit this panic to our logs.
+	cf := util.NewChainedAndFronted(client.Addr)
+	configFetcher := config.NewFetcher(settings.GetUserID, settings.GetToken, cf)
```

Comment:
The UserID is generated by the pro server with `POST /user-create`, this is currently triggered by the Pro UI. Free users also have need to have an UserID, that's how we can assign them a referral code.